airflow/datasets/manager.py

```diff
@@ -71,13 +76,14 @@ def register_dataset_change(
         For local datasets, look them up, record the dataset event, queue dagruns, and broadcast
         the dataset event
         """
+        # todo: add test so that all usages of internal_api_call are added to rpc endpoint
```

Comment:
Not sure if you want to do that later or you forgot this todo, just calling it out here :)

Reply:
it's a "do later" :) just like an idea